janet/src/main/java/io/techery/janet/helper/ActionStateSubscriber.java

```diff
@@ -29,6 +33,11 @@
         return this;
     }
 
+    public ActionStateSubscriber<A> onFinish(Action0 onFinish) {
```

Comment:
why is `Action0` and not `Action1<A>`? I believe it would be more helpful and consistent if one knows action he listens finish for